tchannel/tornado/connection.py

```diff
@@ -205,50 +211,64 @@ def _loop(self):
         #
         # Must be started only after the handshake has been performed.
         self._loop_running = True
+
         while not self.closed:
-            message = yield self._recv()
-            # TODO: There should probably be a try-catch on the yield.
+            try:
+                message = yield self._recv()
+            except tornado.iostream.StreamClosedError:
+                log.warning("Stream has been closed.")
```

Comment:
This is a useless warning. Drop it. If the stream is closed, that means that the connection should be terminated. We don't need to treat that as a warning.